molecule/forget-package/converge.yml

```diff
@@ -16,6 +16,6 @@
   hosts: all
   become: true
   roles:
-    - role: vexxhost.kubernetes.forget_package
+    - role: vexxhost.containers.forget_package
```

Comment:
this molecule test should be moved to the collections repo

Reply:
by this i mean the entire `molecule/forget-package` folder should be moved to the containers collection, and then the tests should be there, and so we dont have to test it here :)